mavedb_mapping/blat_alignment.py

```diff
@@ -5,7 +5,7 @@
 import subprocess
 from gene.database import create_db
 
-qh = QueryHandler(create_db())
+qh = QueryHandler(create_db("postgres://postgres@localhost:5432/gene_normalizer"))
```

Comment:
Hi, I accidentally used `"postgres://postgres@localhost:5432/gene_normalizer"` here. I apologize for the oversight in the code. Initially, I had some issues with gene-normalizer, which have now been resolved. When I was facing those issues I had to hardcode a localhost instance of postgres  to use gene normalize.

Reply:
Yeah, I'd echo Alex's suggestion here -- it's best to avoid hardcoding file paths and service URLs in applications, because that prevents downstream users from configuring those values themselves. For the gene normalizer, if you don't pass any arguments to this method, users can set the endpoint themselves using the `GENE_NORM_DB_URL` environment variable. Full behavior is described [here](https://gene-normalizer.readthedocs.io/en/latest/usage.html#internal-python-api).